[ADD] MRP: using work centers (#3530)
odoo/documentation: content/applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

manufacturing order is created for a product, any work orders listed in the :guilabel:`Operations`
tab of the product bill of materials (BoM) will be automatically created as well and assigned to the
specified work center. Work orders can be managed in the :guilabel:`Manufacturing` module by
selecting :menuselection:`Operations --> Work Orders`.

In order to use work centers, the :guilabel:`Work Orders` feature must first be enabled. To do so,
go to the :guilabel:`Manufacturing` module, select :menuselection:`Configuration --> Settings`, and
activate the checkbox next to :guilabel:`Work Orders`. Work centers can then be created and managed
by selecting :menuselection:`Configuration --> Work Centers`.

Create a work center
====================
Within the :guilabel:`Manufacturing` module, select
:menuselection:`Configuration --> Work Centers --> Create`. The work center form can then be filled
out as follows:

- :guilabel:`Work Center Name`: give the work center a concise name that describes the type of
  operations it will be used for
- :guilabel:`Alternative Workcenters`: specify an alternative work center for operations to be

>   carried out at if the main work center is not available

Reviewer: ```suggestion
  carried out as if the main work center is not available
```

- :guilabel:`Code`: assign the work center a reference code
- :guilabel:`Working Hours`: define the number of hours that the work center can be in use each week
- :guilabel:`Company`: select the company that the work center belongs to

.. image:: using_work_centers/work-center-form.png
   :align: center

>    :alt: An example of a fully configured work center form.

Reviewer: Comment

   :align: center
   :alt: The general information tab of the work center form.

Assign equipment to a work center
---------------------------------
Using the :guilabel:`Equipment` tab, it is possible for specific pieces of equipment to be assigned
to a work center. The following information will be displayed for each piece of equipment added:

- :guilabel:`Equipment Name`: the name of the piece of equipment
- :guilabel:`Technician`: the technician responsible for servicing the equipment
- :guilabel:`Equipment Category`: the category the equipment belongs to
- :guilabel:`MTBF`: mean time between failures; the average time that the piece of equipment will
  operate before failing
- :guilabel:`MTTR`: mean time to recovery; the average time it takes for the equipment to become
  fully operational again
- :guilabel:`Est. Next Failure`: an estimate of when the next equipment failure will occur

.. image:: using_work_centers/work-center-equipment.png
   :align: center
   :alt: The equipment tab of the work center form.

.. note::
    :guilabel:`MTBF`, :guilabel:`MTTR`, and :guilabel:`Est. Next Failure` are all calculated
    automatically based on past failure data, if any exists.

> Integrate IoT devices

Reviewer: ```suggestion
Integrate :abbr:`IoT (Internet of Things)` devices
```

Using the :guilabel:`Equipment` tab, it is possible for specific pieces of equipment to be assigned
to a work center. The following information will be displayed for each piece of equipment added:

- :guilabel:`Equipment Name`: the name of the piece of equipment
- :guilabel:`Technician`: the technician responsible for servicing the equipment
- :guilabel:`Equipment Category`: the category the equipment belongs to
- :guilabel:`MTBF`: mean time between failures; the average time that the piece of equipment will
  operate before failing
- :guilabel:`MTTR`: mean time to recovery; the average time it takes for the equipment to become
  fully operational again
- :guilabel:`Est. Next Failure`: an estimate of when the next equipment failure will occur

.. image:: using_work_centers/work-center-equipment.png
   :align: center
   :alt: The equipment tab of the work center form.

.. note::
    :guilabel:`MTBF`, :guilabel:`MTTR`, and :guilabel:`Est. Next Failure` are all calculated
    automatically based on past failure data, if any exists.

Integrate IoT devices
---------------------
The :guilabel:`IoT Triggers` tab enables the integration of :abbr:`IoT (Internet of Things)` devices

> with a work center:

Reviewer 2: Too lean.

What is IoT? Why do we care? How does IoT integrate with Odoo work centers? 

Are there any best-practice IoT configurations we can mention here (can we hook up sensors to detect gas leaks, for example)? What's the example you used in the screenshot...maybe explain that one?

- :guilabel:`Device`: specifies the IoT device to be triggered
- :guilabel:`Key`: the security key for the device
- :guilabel:`Action`: the IoT device action triggered

.. image:: using_work_centers/work-center-iot.png
   :align: center
   :alt: The IoT Triggers tab of the work center form.

> Use case: configure an alternative work center
> ==============================================

Reviewer 2: Not sure `Use Case:` is needed 🤔 

```suggestion
Route surplus work orders to an alternative work center
=======================================================
```

When a work center is at capacity, it cannot accept any new work orders. Instead of waiting for the
work center to become available, it is possible to specify an alternative work center where surplus
work orders should be carried out.

> Begin by creating a new work center. Configure the :guilabel:`Equipment` tab so that it has all of
> the same equipment as the main work center. This will ensure that the same tasks can be carried out
> at both work centers. Navigate to the main work center and include the new work center in the
> :guilabel:`Alternative Workcenters` selection field.
> 
> Now, create a new manufacturing order that uses the main work center for one of its operations. The
> main work center will automatically be selected for the operation in the :guilabel:`Work Orders`
> tab. After confirming the manufacturing order, click the :guilabel:`Plan` button that appears at the
> top left of the form.

Reviewer 2: Added more instruction/context and separated the content with both line breaks and time-order language so it's easier to consume + pacing is easier to follow.

Now, if a reader came to this doc and didn't read the instructions before this section (which is very common since most doc reader skim), they should still be able to accomplish their desired outcome all the same 😉 

This section tbh is really a good example of what we're looking for, as it's narrative driven, and explains the UI _while_ giving instructions at the same time. Instead of List --> List --> List --> Use Case, every section like this could stand on it's own as an explicit, self-contained use case.

```suggestion
Begin by creating a new work center by navigating to :guilabel:`Manufacturing --> Configuration -->
Work Centers` and filling out the work center form, using the instructions above as a guide.

Next, fill out the :guilabel:`Equipment` tab so that it has all of the same equipment as the main
work center, which will ensure that the same tasks can be carried out at both work centers.

Then, navigate to the main work center and include the new work center in the :guilabel:`Alternative
Workcenters` selection field.

With the alternative work center configuration complete, create a new manufacturing order that uses
the main work center for one of its operations. The main work center will automatically be selected
for the operation in the :guilabel:`Work Orders` tab. After confirming the manufacturing order,
click the :guilabel:`Plan` button that appears at the top left of the form.
```

.. image:: using_work_centers/manufacturing-order-plan-button.png
   :align: center
   :alt: Click the plan button to automatically select an available work center.

If the main work center is at capacity, the work center selected for the operation will be
automatically changed to the alternative work center.

.. image:: using_work_centers/automatic-work-center-selection.png
   :align: center
   :alt: The alternative work center is automatically selected.

> Monitor work center performance
> ===============================

Reviewer 2: ```suggestion
Monitor work center performance
===============================

```

Performance for an individual work center can be viewed by selecting
:menuselection:`Configuration --> Work Centers`, and clicking on a work center. A variety of metrics

> showing work center performance can be viewed at the top right of the form:

Reviewer 2: Can go deeper here on work center reporting?
This stuff really matters to shop floor managers...

=====================================
Manage work orders using work centers
=====================================

Odoo Manufacturing allows for work orders to be carried out at specific work centers. When a
manufacturing order is created for a product, any work orders listed in the :guilabel:`Operations`
tab of the product bill of materials (BoM) will be automatically created as well and assigned to the
specified work center. Work orders can be managed in the :guilabel:`Manufacturing` module by
selecting :menuselection:`Operations --> Work Orders`.

In order to use work centers, the :guilabel:`Work Orders` feature must first be enabled. To do so,
go to the :guilabel:`Manufacturing` module, select :menuselection:`Configuration --> Settings`, and
activate the checkbox next to :guilabel:`Work Orders`. Work centers can then be created and managed
by selecting :menuselection:`Configuration --> Work Centers`.

Create a work center
====================

> Within the :guilabel:`Manufacturing` module, select
> :menuselection:`Configuration --> Work Centers --> Create`. The work center form can then be filled
> out as follows:

Reviewer 2: Can break the tag here.
```suggestion
Within the :guilabel:`Manufacturing` module, select :menuselection:`Configuration --> Work Centers
--> Create`. The work center form can then be filled out as follows:
```
